src/components/VList/VListTile.js

```diff
@@ -70,7 +70,7 @@ export default {
 
     data.attrs = Object.assign({}, data.attrs, this.$attrs)
 
-    return h('li', {
+    return h('div', {
```

Comment:
Is this still needed? It was only there to be valid HTML afaik, the root element can just be `.list__tile` now.

Reply:
Let's not get into a minor refactor. The goal is to simply change the tags for now.

Reply:
k